articles/architecture-scenarios/_includes/_operations/_introduction.md

```diff
@@ -11,9 +11,16 @@ Establishing effective operational behaviors is something that successful custom
 *	Can I scan Auth0 logs to determine if peak loads in my application trigger any rate limits or other errors?
 *	What email services should I be using to support production volumes of email messages to users? Why can’t I use Auth0's out-of-box email provider in my production environment?
 *	Why would I need to configure my firewall, and what firewall ports will I need to open for internal services that need to receive communications from Auth0 (such as custom databases, web services, and email servers)?
+<% if (platform === "b2b") { %>
+* Will you want to provide self-service provisioning of your customer organization IDPs?
```

Comment:
I can only seem to comment here (perhaps because the PR had been merged and closed?), but for consistency I think we should follow the "first person" strategy adopted in all the other bullet points as opposed to switching to "second person" (just for this one).

Reply:
Added to separate PR